.github/workflows/build.yaml

```diff
@@ -543,6 +543,49 @@ jobs:
           name: book.pdf
           path: ${{github.workspace}}/docs/books/user-guide/book.pdf
 
+  python:
+    name: 'Python Checkers (${{ matrix.container }})'
+    runs-on: '${{ matrix.os }}'
+    strategy:
+      matrix:
+        os: [ 'ubuntu-20.04' ]
+        container: [ 'fedora' ]
+        containerTag: [ '35' ]
+
+    container:
+      image: 'library/${{ matrix.container }}:${{ matrix.containerTag }}'
+      volumes:
+        - ${{github.workspace}}:${{github.workspace}}
+
+    env:
+      DispatchBuildDir: ${{github.workspace}}/build
+      InstallPrefix: ${{github.workspace}}/install
+      DispatchCMakeExtraArgs: >
+        -GNinja
+        -DCONSOLE_INSTALL=OFF
+
+    steps:
+
+      - name: Install build dependencies
+        run: |
+          dnf install -y 'dnf-command(builddep)' && dnf builddep -y qpid-dispatch-router && dnf install -y git tox ninja-build
```

Comment:
Short and sweet. This is what I had in mind when I filled the aspirational [DISPATCH-1928 Design for a platform-independent way of doing CI](https://issues.apache.org/jira/browse/DISPATCH-1928). The CI script should be short and sweet, avoiding the ever repeating lists of dependencies.

One way to get a nicer CI script is to rely on the RPM shipped with Fedora, which lists build requirements. Not flexible enough, and works only on Fedora, but the YAML is so much nicer for it.